### PR TITLE
fix: pin ts-patch for api build

### PR DESCRIPTION
**Pull request by lucasmarshall**

### CODE REVIEW

File: apps/api/Dockerfile [hunk 1]
```diff
@@ -23,7 +23,7 @@ COPY turbo.json turbo.json
 COPY tsconfig.json tsconfig.json
 
 RUN --mount=type=cache,mode=0777,target=/app/.yarn/cache yarn workspace @supaglue/db prisma generate
-RUN --mount=type=cache,mode=0777,target=/app/.yarn/cache yarn workspace api dlx ts-patch install -s
+RUN --mount=type=cache,mode=0777,target=/app/.yarn/cache yarn workspace api dlx -p ts-patch@2.1.0 ts-patch install -s
```
Comment (lucasmarshall) [inline apps/api/Dockerfile:26]: 3.0 breaks

File: apps/salesforce-pub-sub/Dockerfile [hunk 2]
```diff
@@ -23,7 +23,6 @@ COPY turbo.json turbo.json
 COPY tsconfig.json tsconfig.json
 
 RUN --mount=type=cache,mode=0777,target=/app/.yarn/cache yarn workspace @supaglue/db prisma generate
-RUN --mount=type=cache,mode=0777,target=/app/.yarn/cache yarn workspace salesforce-pub-sub dlx ts-patch install -s
```
Comment (lucasmarshall) [inline apps/salesforce-pub-sub/Dockerfile:26]: Not used in this service

File: apps/api/Dockerfile (hunk 1 above)
Comment (albertyfwu) [reply apps/api/Dockerfile:26]: what's the way to make this work with 3.0?

**Review by albertyfwu — APPROVED** [review]

File: apps/api/Dockerfile (hunk 1 above)
Comment (lucasmarshall) [reply apps/api/Dockerfile:26]: https://github.com/supaglue-labs/supaglue/pull/1003

**Pull request merged**